velox/exec/tests/SpillerTest.cpp

```diff
@@ -269,12 +269,19 @@ class SpillerTest : public exec::test::RowContainerTestBase {
     VELOX_ASSERT_THROW(spiller_->spill(RowContainerIterator{}), "");
 
     verifySortedSpillData(&spillPartition, outputBatchSize);
+    auto lockStats = spillStats_.rlock();
+    common::updateGlobalSpillReadBytes(lockStats->spillReadBytes);
+    common::updateGlobalSpillReads(lockStats->spillReads);
+    common::updateGlobalSpillReadTime(lockStats->spillReadTimeUs);
+    common::updateGlobalSpillDeserializationTimeUs(
+        lockStats->spillDeserializationTimeUs);
```

Comment:
Curious why are we doing this?

Reply:
Only the write stats would be updated to the Global stats in the spill finish method. We could only update the read part in the Operator::close, so we have to update the read stats manually here.

Reply:
Can you call global methods to update on the read path? Thanks!